debian/docker/Dockerfile

```diff
@@ -32,11 +32,8 @@ RUN apt-get update -y \
  && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC \
     apt-get install --no-install-recommends -y \
     wget debhelper gnupg devscripts build-essential mmv lsb-release \
-    libssl*.* liblttng-ust* \
-    libfontconfig*-dev libcurl*openssl-dev libfreetype*-dev libssl-dev \
- && apt-get clean autoclean -y \
- && apt-get autoremove -y \
- && rm -rf /var/lib/apt/lists/*
+    libssl*.* liblttng-ust* libfontconfig*-dev libcurl*openssl-dev \
```

Comment:
Why remove the cleanup here? The goal is to keep the intervening layers as clean as possible.